### DISCUSSION TITLE
Forward slash in `dag_run_id` makes dag run unfindable in API

### DISCUSSION THREAD

**Discussion by davidavdav** [apache/airflow#19745] [Q&A]
Hello, 

We're using airflow in an event-based setup, where, e.g., media file upload triggers various dags.  We've found it useful to use the media-path `path/to/mediafile` as `dag_run_id`, and this works for the most part in airflow.  However, it seems a specific dag run can no longer be found in the REST API through the URL part `/api/v1/dags/{dag}/dagRuns/path/to/mediafile`, escaping the `/` with `%2f` does not help here.  

It seems not forbidden to have `/` characters in `dag_run_id`, is this true, and should it be that way?

**Comment by potiuk** [comment]
I think you should open an issue for that one.
